dev/benchmarks/microbenchmarks/lib/ui/get_transform_bench.dart

```diff
@@ -0,0 +1,162 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I forget how this works: Does this benchmark need to be registered anywhere so it runs regularly and uploads data to skia perf? Or do we automatically run all benchmarks in `dev/benchmarks/microbenchmarks/lib/ui`?

Reply:
Added to microbenchmarks.dart